TS3AudioBot/CommandSystem/Commands/FunctionCommand.cs

```diff
@@ -246,7 +246,7 @@ private static object GetDefault(Type type)
 		}
 
 		/// <summary>
-		/// A conveniance method to set the amount of required parameters and returns this object.
```

Comment:
i think `convenience` was the word i was looking here for.

Reply:
Oh, I see. I didn't know it was an actual term.